tests/tensorflow/nightly/serving.libsonnet

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local experimental = import '../experimental.libsonnet';
+local common = import 'common.libsonnet';
+local gpus = import 'templates/gpus.libsonnet';
+local mixins = import 'templates/mixins.libsonnet';
+local tpus = import 'templates/tpus.libsonnet';
+
+{
+  local servingTest = common.ServingTest + experimental.TensorflowServingTpuVmMixin {
```

Comment:
Since you use these `common.ServingTest` and `experimental.TensorFlowServingTpuVmMixin` together as a base object, can you combine them into one object?

Reply:
Ack - unclear to me if we will eventually add non TPU VM serving tests.

I think for now I want to keep it like this so I don't forget that it's combined in `common.ServingTest`?